api/db/setup_local_vars.sh

```diff
@@ -1,3 +1,3 @@
 source setup_vars.sh
-WORKBENCH_PASSWORD=workbench!pwd
-
+export WORKBENCH_PASSWORD=workbench!pwd
```

Comment:
Oh, I see: they'll shadow the host variables during the compile/test step, but not during the deploy step.